test/00_init.ts

```diff
@@ -42,27 +42,29 @@ Bluebird.resolve(fs.promises.readdir(__dirname))
 			throw new Error(`Prefix ${prefix} has already been used`);
 		}
 		prefixes[prefix] = true;
```

Comment:
Do we need this v like we did elsewhere?
```suggestion
		prefixes[prefix] = true;
		if (prefix === '00') {
			// Don't double load this file
			return;
		}
```

Reply:
Yeah, just because it's not expected to be a test file really so doesn't export (or need to) an init function